main.tf

```diff
@@ -145,7 +145,7 @@ data "template_cloudinit_config" "nfs" {
 }
 
 module "nfs" {
-  source                         = "./modules/azurerm_vm"
+  source = "./modules/azurerm_vm"
```

Comment:
Don't lint the files. There are several issues/prs where you're run `tf format` or something similar and it creates a lot of changes. Formatting should be done by hand. This creates lots of noise in the issue/pr with changes.

Reply:
I'd suggest otherwise - i.e., require 'terraform fmt` prior to all commits. It uses Terraform formatting Vs. custom/editor formatting that helps with readability of the code base.